plugins/by-name/vim-suda/default.nix

```diff
@@ -0,0 +1,6 @@
+{ lib, ... }:
+lib.nixvim.vim-plugin.mkVimPlugin {
+  name = "vim-suda";
+  globalPrefix = "suda#";
```

Comment:
Looking at https://github.com/lambdalisue/vim-suda/blob/b97fab52f9cdeabe2bbb5eb98d82356899f30829/doc/suda.txt#L171-L194, most globals are using the `suda#` prefix, but there is also `suda_smart_edit` which uses `suda_`...

Kinda frustrating, but I guess this is fine?

Reply:
I've added a pull request to add fix the mismatched option prefix. https://github.com/lambdalisue/vim-suda/pull/84